tests/install_tests.rs

```diff
@@ -336,3 +336,16 @@ async fn test_channels_changed() {
     let lock_file = pixi.up_to_date_lock_file().await.unwrap();
     assert!(lock_file.contains_match_spec(DEFAULT_ENVIRONMENT_NAME, platform, "bar ==1"));
 }
+
+#[tokio::test]
```

Comment:
```suggestion
#[tokio::test(flavor = "multi_thread", worker_threads = 1)]
#[serial]
#[cfg_attr(not(feature = "slow_integration_tests"), ignore)]
```
Actually installing is known to be flaky and slow. So we add these to prevent them from being run in the pre-push hooks and avoid flackiness.